multipart_file_upload.rb

```diff
@@ -245,61 +245,58 @@ def complete_multipart_file_upload(parts, s3_key, upload_id)
 # was previously uploaded to Sentera's cloud storage with one
 # of the mutations in Sentera's GraphQL API that accepts a
 # file ID as an input. In this example, we'll use the
-# import_files GraphQL mutation, and attach the file to
-# a feature set that the caller is permitted to access.
+# import_mosaic GraphQL mutation to attach the file to
+# the mosaic created in step 1.
 #
+# @param [string] owner_sentera_id Sentera ID of the resource in
+#                                  FieldAgent to which the file should
+#                                  be attached.
+# @param [string] parent_sentera_id Sentera ID of the resource in
+#                                   FieldAgent that will is the parent
+#                                   of the owner.
 # @param [string] file_id ID of the uploaded file
-# @param [string] file_owner_type Type of file owner to create. For example,
-#                                 FEATURE_SET, MOSAIC, etc.
-# @param [string] file_owner_sentera_id Sentera ID of the resource
-#                                       (field, survey, feature set, etc.)
-#                                       to which the file should be attached.
-# @param [string] file_path Fully qualified path to file that was uploaded
 #
 # @return [Hash] Hash containing results of the GraphQL request
 #
-def use_file(file_id, file_owner_type, file_owner_sentera_id, file_path)
+def use_file(owner_sentera_id, parent_sentera_id, file_id)
```

Comment:
```suggestion
def import_mosaic(owner_sentera_id, parent_sentera_id, file_id)
```
Same idea as above - I think these are easier to chunk/think about as the mutation inside them.